USB: cdc-acm: don't set tty low_latency flag

"The hack setting of it for high data rate devices was obsolete when the
tty buffer changes went in so it can definitely go in those cases."
  --Alan Cox

Signed-off-by: Chuck Ebbert <cebbert@redhat.com>

Index: linux-2.6.29.noarch/drivers/usb/class/cdc-acm.c
===================================================================
--- linux-2.6.29.noarch.orig/drivers/usb/class/cdc-acm.c
+++ linux-2.6.29.noarch/drivers/usb/class/cdc-acm.c
@@ -546,10 +546,6 @@ static int acm_tty_open(struct tty_struc
 	tty->driver_data = acm;
 	acm->tty = tty;
 
-	/* force low_latency on so that our tty_push actually forces the data through,
-	   otherwise it is scheduled, and with high data rates data can get lost. */
-	tty->low_latency = 1;
-
 	if (usb_autopm_get_interface(acm->control) < 0)
 		goto early_bail;
 	else
